src/diagnostic/diagnostic_model_view.hpp

```diff
@@ -37,7 +37,8 @@ class ModelView : public IModelView
     using GridLayout = typename Model::gridlayout_type;
     using PatchProperties
```

Comment:
it's possible we want to have a separate dict for patch properties, and for global properties
as now the patch properties has no need for "std::vector<string>", and the mpi attribute write function per mpi does not support it